gherkin-utils/javascript/src/pretty.ts

```diff
@@ -1,29 +1,31 @@
 import * as messages from '@cucumber/messages'
 
-export default function pretty(gherkinDocument: messages.GherkinDocument): string {
+type Syntax = 'markdown' | 'gherkin'
+
+export default function pretty(gherkinDocument: messages.GherkinDocument, syntax: Syntax): string {
   const feature = gherkinDocument.feature
-  let s = prettyTags(feature.tags)
+  let s = prettyTags(feature.tags, 0, syntax)
```

Comment:
Maybe we could rename that `s` variable in the process?

Reply:
Agreed

Reply:
`prettifiedFeature`?